### ISSUE TITLE
[Feature Request] Toggle Embeds on/off on Combine Emebeds

### ISSUE THREAD

**Issue by Mattssn** [cubiq/ComfyUI_IPAdapter_plus#329]
Hello! I love using IP adapters and yours have been pretty damn solid, I am requesting some on/off toggles on Combine Embed nodes, My workflow uses 4 images for ip adapter but I don't always like to use all 4, I was using nodes that allowed me to toggle off each IP adapter previously, but stopped working with the update.

![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/54347017/792a23bc-7de6-455a-9d8c-f03a6608b902)

Would it be possible to add an on/off toggle on each input into the Combine Embeds node to allow only using 1-4 images for the generation?

Thanks!

**Comment by danton267** [comment]
Why not simply bypass the input embed nodes if you don't want to use them?

@cubiq do you want this behaviour? I can push the PR
![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/51248046/d09578af-3898-4ea6-b1e5-936443e56dfe)
![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/51248046/25e47414-42d4-4e6d-ae44-ff090a01bbd9)

**Comment by cubiq** [comment]
no, this is outside of the scope of the node. you can just enable/disable the embeds node if you want to trigger them

**Comment by Mattssn** [comment]
> Why not simply bypass the input embed nodes if you don't want to use them?
> 
> @cubiq do you want this behaviour? I can push the PR ![image](https://private-user-images.githubusercontent.com/51248046/318245970-d09578af-3898-4ea6-b1e5-936443e56dfe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNzQ1ODEsIm5iZiI6MTcxMjA3NDI4MSwicGF0aCI6Ii81MTI0ODA0Ni8zMTgyNDU5NzAtZDA5NTc4YWYtMzg5OC00ZWE2LWIxZTUtOTM2NDQzZTU2ZGZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDE2MTEyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM4ZWM0Zjg5YTM4NWJiZDM4NzRmYjhhN2UyYmQ2YzZhN2ZhYWY0MWI5ZDk5OWY5ZDQwNjc3YTY0YzY0MzU2NDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6QLlPGtXg0vgU6rZlqNODisHy79Uj5BiUlh0vRBeF2k) ![image](https://private-user-images.githubusercontent.com/51248046/318246036-25e47414-42d4-4e6d-ae44-ff090a01bbd9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNzQ1ODEsIm5iZiI6MTcxMjA3NDI4MSwicGF0aCI6Ii81MTI0ODA0Ni8zMTgyNDYwMzYtMjVlNDc0MTQtNDJkNC00ZTZkLWFlNDQtZmYwOTBhMDFiYmQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDE2MTEyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlOWIzYzkzYTc5NjgyMTczYzAyMjc3NWNlZDQ1MjQxNDZkNTgzZTNlYzlhNGNhMzRmOGVmYzdiYTI1YzY4MTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3McDMQdJ4r4vo0jzz-lKP9dhFxmLMpohyR7DuWtrgwQ)

Have you tried this by chance, if I try this, it complains that there are missing embed inputs, it seems that if I have 5 embeds and I mute embed 2 and 4, it throws an error because I have no input on 2, if I enable 2 it would then complain I am missing input on 4, hope I explained this well enough.

**Comment by danton267** [comment]
I can bypass any number of them, they can be at any place, and I can also disconnect any of them. The code simply filters them out.
<img src="https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/51248046/dd9c2044-d94c-4eec-a7d0-fd8b5d8bef02" width="50%"><img src="https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/51248046/eb595c36-61c5-4cee-aa48-72353976abdc" width="50%">


Looking back at the method history, there are no changes between the versions (after rework), so even if you do not have the latest package version it should not be a problem..

Not sure what it could be. Maybe you have an outdated ComfyUI and they changed how the bypass nodes are being propagated? Although that sounds unlikely. Still, try updating everything.
